administration/console.md

```diff
@@ -132,6 +132,9 @@ The pertinent files controlling console settings are stored under `$OPENHAB_USER
 
 The exact locations of these files will vary based on your platform and installation method, e.g. `/var/lib/openhab2/etc/` or `openhab2/userdata/etc/`.
 
+Be aware that the these files may get overwritten when upgrading openHAB, so to add custom parameters or overwrite the default values, you can change the configuration file `services/runtime.cfg` in
+the openHAB configuration directory instead, e.g. `/etc/openhab2/services/runtime.cfg`.
```

Comment:
Please consider changing as follows:
"...you can change the configuration file `runtime.cfg` in the `$OPENHAB_CFG/services/` directory instead.

Reply:
It should be `$OPENHAB_CONF/services`.

Reply:
@BClark09 is correct, of course...